docs/js/links-in-new-tabs.js

```diff
@@ -0,0 +1,10 @@
+// This takes any external links and adds _blank to make sure they open in a new tab. Links to other
+// pages within the documentation will remain in the same tab
+// Source: https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown#answer-4425214
+
+var links = document.links;
```

Comment:
Missing copyright.  Also, we usually use `const` instead of `var`

Reply:
Thanks, I have updated these.